tracing-tracy/src/lib.rs

```diff
@@ -196,19 +162,28 @@ thread_local! {
     static CACHE: StrCache = const { StrCache::new() };
 }
 
-impl<S, F> Layer<S> for TracyLayer<F>
+impl<S, C> Layer<S> for TracyLayer<C>
 where
     S: Subscriber + for<'a> registry::LookupSpan<'a>,
-    F: for<'writer> FormatFields<'writer> + 'static,
+    C: Config + 'static,
 {
     fn on_new_span(&self, attrs: &Attributes<'_>, id: &Id, ctx: Context<'_, S>) {
         let Some(span) = ctx.span(id) else { return };
 
         let mut extensions = span.extensions_mut();
-        if extensions.get_mut::<FormattedFields<F>>().is_none() {
-            let mut fields =
-                FormattedFields::<F>::new(CACHE.with(|cache| cache.acquire().into_inner()));
-            if self.fmt.format_fields(fields.as_writer(), attrs).is_ok() {
+        if extensions
+            .get_mut::<FormattedFields<C::Formatter>>()
```

Comment:
Are type alises inside impls a thing? Would be nice for `FormattedFields<C::Formatter>`.